.github/workflows/test.yml

```diff
@@ -142,26 +137,34 @@ jobs:
     runs-on: ubuntu-latest
     container: denoland/deno
     steps:
-      - uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
+      - name: Checkout the repo
+        uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
+
       - name: deno fmt
         run: deno fmt --check
+
       - name: deno lint
         run: deno lint
+
       - name: deno check
         run: deno check --no-lock extensions/**/*.ts
 
   shellcheck:
+    if: github.event_name != 'schedule'
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
+      - name: Checkout the repo
+        uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
+
       - name: Script Style Check
-        if: github.event_name != 'schedule'
-        run: shellcheck -o all -S style -s sh $(find . -iname "*.sh")
+        run: find . -iname "*.sh" -print0 | xargs -0 shellcheck -o all -S style -s sh
 
-  # This job reports the results of the matrixes above
+  # This job reports the results of the test jobs above and is used
+  # to enforce status checks in the repo settings without needing
+  # to update those settings every time the test jobs are updated.
   test:
     if: always()
-    needs: [windows, clippy, test-matrix, all-features, oldstable]
+    needs: [rustfmt, clippy, test-matrix, oldstable, all-features, deno-checks, shellcheck]
```

Comment:
`rustfmt`, `oldstable`, and `deno-checks` are already required by repo settings... There is no reason to include them here (especially in a PR that leaves those jobs unchanged)

Reply:
It has been reverted. Only the jobs with matrices are included in this test rollup job now.